src/main/java/seedu/agendum/model/task/ChildRecurringTask.java

```diff
@@ -0,0 +1,81 @@
+package seedu.agendum.model.task;
+
+import java.time.LocalDateTime;
+import java.util.Optional;
+
+public class ChildRecurringTask extends RecurringTask {
```

Comment:
Yup it does say so on the handouts, but it never says the relationship between these two classes. In my implementation, RecurringTask stores the general information, childRecurringTask stores the unque information, I think it still follows the abstraction-occurrence pattern.